README.md

```diff
@@ -93,6 +93,26 @@ Building clang from sources takes time and resources and it can be avoided:
   moment, see [KhronosGroup/SPIRV-LLVM-Translator#477](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/477)
   to track progress, discuss and contribute.
 
+## Configuring SPIR-V Headers
+
+The translator build is dependent on the official Khronos header file
+`spirv.hpp` that maps SPIR-V extensions, decorations, instructions,
+etc. onto numeric tokens. The official header version is available at
+[KhronosGroup/SPIRV-Headers](https://github.com/KhronosGroup/SPIRV-Headers).
+There are several options for accessing the header file:
+- By default, the header file repository will be downloaded from
+Khronos Group GitHub and put into `<build_dir>/SPIRV-Headers` (for
+out-of-tree builds) or into `llvm/projects` (for in-tree builds).
+- If you are building the translator in-tree, you can manually
+  download the SPIR-V Headers repo into `llvm/projects` - this
+  location will be automatically picked up by the translator build
+  scripts.
```

Comment:
Note here: this is rather would be done by upstream LLVM CMake scripts: for (almost) each directory in `projects`, [`add_llvm_subdirectory`](https://github.com/llvm/llvm-project/blob/main/llvm/cmake/modules/AddLLVM.cmake#L1337) is called (through `add_llvm_external_project`), which defines `LLVM_EXTERNAL_project_name_SOURCE_DIR` which we later use.

`project_name` is generated automatically in this case and we should probably require particular name for the headers subdirectory to make it work

Reply:
Updated.